Documents/C_SHARP.md

```diff
@@ -78,23 +78,23 @@ For example:
 ```csharp
 public class MyClass 
 {
-    public int publicField;
-    private int packagePrivate;
-    private int myPrivate;
-    protected int myProtected;
+    public int PublicField;
+    private int _packagePrivate;
+    private int _myPrivate;
+    protected int _myProtected;
```

Comment:
None of these adhere to the standard described in line 74. It needs to be amended as well, to reflect that public fields will be PascalCase, and private fields will be camelCase with a leading underscore (if that is what the scheme is intended to be).